atom/src/sortedmap.cpp

```diff
@@ -278,17 +277,26 @@ SortedMap_new( PyTypeObject* type, PyObject* args, PyObject* kwargs )
     return self;
 }
 
-
-static void
-SortedMap_clear( SortedMap* self )
-{
-    // Clearing the vector may cause arbitrary side effects on item
-    // decref, including calls into methods which mutate the vector.
-    // To avoid segfaults, first make the vector empty, then let the
-    // destructors run for the old items.
-    SortedMap::Items empty;
-    self->m_items->swap( empty );
-}
+#if PY_MAJOR_VERSION >= 3
+    static int
+    SortedMap_clear( SortedMap* self )
+    {
+        SortedMap::Items empty;
+        self->m_items->swap( empty );
+        return 0;
+    }
+#else
+    static void
+    SortedMap_clear( SortedMap* self )
+    {
+        // Clearing the vector may cause arbitrary side effects on item
+        // decref, including calls into methods which mutate the vector.
+        // To avoid segfaults, first make the vector empty, then let the
+        // destructors run for the old items.
```

Comment:
Maybe move this comment up above the `#if` since it applies to both implementations?